package.json

```diff
@@ -21,37 +21,34 @@
     "lib"
   ],
   "dependencies": {
-    "browser-sync": "^2.9.11",
-    "del": "^2.0.2",
-    "front-matter": "^1.0.0",
-    "gulp-compile-handlebars": "^0.5.0",
-    "gulp-cssnano": "^2.1.2",
-    "gulp-data": "^1.2.0",
-    "gulp-if": "^2.0.0",
-    "gulp-plumber": "^1.1.0",
-    "gulp-postcss": "^7.0.0",
-    "gulp-rename": "^1.2.2",
-    "gulp-util": "^3.0.6",
-    "gulp-watch": "^4.3.11",
-    "lodash": "^3.10.1",
+    "browser-sync": "^2.24.4",
+    "del": "^3.0.0",
+    "front-matter": "^2.3.0",
+    "gulp-compile-handlebars": "^0.6.1",
+    "gulp-cssnano": "^2.1.3",
+    "gulp-data": "^1.3.1",
+    "gulp-if": "^2.0.2",
+    "gulp-plumber": "^1.2.0",
+    "gulp-postcss": "^7.0.1",
+    "gulp-rename": "^1.3.0",
+    "gulp-util": "^3.0.8",
+    "gulp-watch": "^5.0.0",
+    "lodash": "^4.17.10",
     "postcss-class-prefix": "^0.3.0",
-    "postcss-cssnext": "^2.1.0",
-    "postcss-discard-comments": "^2.0.4",
-    "postcss-discard-empty": "^2.0.1",
-    "postcss-foft-classes": "^1.0.2",
-    "postcss-import": "^11.0.0",
-    "postcss-mixins": "^4.0.1",
-    "postcss-use": "^2.3.0",
-    "require-dir": "^0.3.2",
-    "run-sequence": "^1.2.2",
-    "snyk": "^1.30.1",
+    "postcss-foft-classes": "^1.0.4",
+    "postcss-import": "^11.1.0",
+    "postcss-use": "^3.0.0",
+    "require-dir": "^1.0.0",
+    "run-sequence": "^2.2.1",
+    "snyk": "^1.83.0",
     "webpack": "^1.12.2"
   },
   "devDependencies": {
-    "gulp": "^3.9.0",
+    "gulp": "^4.0.0",
```

Comment:
Do we _want_ to bump to Gulp 4 now? I thought there were some big changes and it wasn't stable yet? 🤔 

/CC @gerardo-rodriguez